src/rules/each-key-value-single-line/README.md

```diff
@@ -0,0 +1,48 @@
+# each-key-value-single-line
```

Comment:
Also this rule should follow the naming conventions from stylelint, e.g. `at-rule-each-*`:
https://github.com/stylelint/stylelint/blob/master/docs/user-guide/rules.md#at-rule-1